### DISCUSSION TITLE
Timer1.attachInterrupt(do_isr);

### DISCUSSION THREAD

**Discussion by jafal633** [sstaub/TickTwo#2] [General]
hi  sir

can we us this library  same as Timer1

and how to use it like this  with  ESP8266
Timer1.initialize(    );
Timer1.attachInterrupt(do_isr);

thanks in advance

**Comment by sstaub** [comment]
No, this library is not based on timers, it use millis() and micros().

**Reply by jafal633** [reply]
thank you for your reply  sir
